macro/src/expand.rs

```diff
@@ -1678,18 +1678,18 @@ fn expand_rust_box(
         .explicit_impl
         .map_or(key.end_span, |explicit| explicit.brace_token.span.join());
     let unsafe_token = format_ident!("unsafe", span = begin_span);
-    let prevent_unwind_drop_label = format!("::{} as Drop>::drop", ident);
+    let prevent_unwind_drop_label = quote! { #inner }.to_string();
```

Comment:
This is not an adequate replacement for what was here before.

```cpp
#pragma once
#include <rust/cxx.h>

class Thing;

inline void take_thing(rust::Box<Thing>) {}
```

```rust
#[cxx::bridge]
mod ffi {
    extern "Rust" {
        type Thing;
    }
    unsafe extern "C++" {
        include!("example/include/header.h");
        fn take_thing(_: Box<Thing>);
    }
}

struct Thing;

impl Drop for Thing {
    fn drop(&mut self) {
        panic!("...");
    }
}

fn main() {
    ffi::take_thing(Box::new(Thing));
}
```

Before:

```console
panic in ffi function <example::ffi::Thing as Drop>::drop, aborting.
```

After:

```console
panic in ffi function <example::ffiThing, aborting.
```